docs/content/docs/testing/telemetry.md

```diff
@@ -130,8 +130,9 @@ either tagged (`ITaggedTelemetryPropertyType`) or untagged (`TelemetryEventPrope
 
 ### Understanding Tags
 
-Tags are strings used to classify the properties on telemetry events. In the course of operation,
-the Fluid Framework may emit events with tagged properties, so implementations of `ITelemetryBaseLogger` must be
+Tags are strings used to classify the properties on telemetry events. By default, telemetry properties are untagged
+and untagged properties can be considered safe for general logging. However,
```

Comment:
Not sure if I need some disclaimer here about what "safe for general logging" means.  Internally we have that whole data taxonomy thing, but that's internal so can't really depend on/use the language from there.

@jbissat - SIde note - this is a conversation we should probably have, at least in terms of internal guidance to the team as to what is safe to log (especially given how it can change as recently discussed).